colorcoder/ColorUtils.java

```diff
@@ -0,0 +1,44 @@
+package colorcoder;
+
+public class ColorUtils {
+
+	static ColorPair GetColorFromPairNumber(int pairNumber) {
+		int zeroBasedPairNumber = pairNumber - 1;
+		MajorColor majorColor = MajorColor.fromIndex(zeroBasedPairNumber / Constants.numberOfMinorColors);
+		MinorColor minorColor = MinorColor.fromIndex(zeroBasedPairNumber % Constants.numberOfMinorColors);
+		return new ColorPair(majorColor, minorColor);
+	}
+
+	static int GetPairNumberFromColor(MajorColor major, MinorColor minor) {
+		return major.getIndex() * Constants.numberOfMinorColors + minor.getIndex() + 1;
+	}
+
+	static void testNumberToPair(int pairNumber, MajorColor expectedMajor, MinorColor expectedMinor) {
```

Comment:
test code is bundled along with the production code. Please separate the test code from production code.
Also Utils is a very generic name. generally there is a tendency to put all utility functions to this file, eventhough they might not be related !! Hence try to avoid if possible.